README.md

```diff
@@ -11,6 +11,12 @@ A chat bot for [Slack](https://slack.com) inspired by [llimllib/limbo](https://g
 * Python3 Support
 * [Full-fledged functional tests](tests/functional/test_functional.py)
 
+## Index:
+1. [Installation](#installation)
+1. [Usage](#usage)
+1. [Create custom plugins](#create-plugins)
+1. [Discussion](#discussion)
+
```

Comment:
I would suggest move the change to the documents into a separate PR, this way it's easier for code reviewing and each PR can get merged faster.